tests/socketcall_32bit/Makefile

```diff
@@ -0,0 +1,11 @@
+TARGETS=$(patsubst %.c,%,$(wildcard *.c))
+
+LDLIBS += -lpthread
+
+all: $(TARGETS)
+
+conn: conn.c
+	$(CC) $(CFLAGS) -o $@ $^
```

Comment:
I wondered how that was determined, maybe by the selection of specific 32-bit system calls?  '-m32' sounds like what I wanted, to be certain it was in fact a 32-bit binary.